engine/dlib/src/dlib/buffer.cpp

```diff
@@ -616,4 +621,70 @@ namespace dmBuffer
         buffer->m_ContentVersion++;
         return RESULT_OK;
     }
+
+
+    Result UpdateBounds(HBuffer hbuffer, dmhash_t stream_name) {
```

Comment:
I'd rather see this in 3 parts.
1) Get the data array and length
2) Calculate the bounding min + max from the data array
3) Set the bounding min+max on the buffer

Reply:
You mean better separate the function implementation into three sections, right ?

Or have three separate functions ?

Reply:
Let's discuss on the updated version of the code. Many things have changed.